Description: correct stringprep app name
 The stringprep library was renamed in 1.0.2,
 this patch compensates for that.
Author: Philipp Huebner <debalance@debian.org>

Index: ejabberd/src/ejabberd_app.erl
===================================================================
--- ejabberd.orig/src/ejabberd_app.erl
+++ ejabberd/src/ejabberd_app.erl
@@ -255,7 +255,7 @@ start_apps() ->
     ejabberd:start_app(p1_yaml),
     ejabberd:start_app(p1_tls),
     ejabberd:start_app(p1_xml),
-    ejabberd:start_app(p1_stringprep),
+    ejabberd:start_app(stringprep),
     ejabberd:start_app(p1_zlib),
     ejabberd:start_app(cache_tab).
 
